api/clients/v2/disperser_client.go

```diff
@@ -16,21 +16,21 @@ import (
 	"google.golang.org/grpc"
 )
 
-type DisperserClientV2Config struct {
+type DisperserClientConfig struct {
 	Hostname          string
 	Port              string
 	UseSecureGrpcFlag bool
 }
 
-type DisperserClientV2 interface {
+type DisperserClient interface {
```

Comment:
If we keep the package name clients, then we could simplify these to `clients.DisperserConfig` and `clients.Disperser`.